Containerfile.nvidia

```diff
@@ -48,7 +48,7 @@ RUN --mount=type=cache,dst=/var/cache/dnf \
     ; else \
        export KERNEL_NAME="kernel-surface" \
     ; fi && \
-    /tmp/build-kmod-nvidia.sh 550 && \
+    /tmp/build-kmod-nvidia.sh closed && \
```

Comment:
We can have this be a variable unless you just want to have this one word change for simplicity.